src/components/calcite-slider/calcite-slider.e2e.ts

```diff
@@ -148,7 +151,7 @@ describe("calcite-slider", () => {
     expect(ticks.length).toBe(11);
   });
 
-  it("fires calciteSliderChange event on changes", async () => {
+  it("keyboard interaction : key press should change the value and emit input and change events", async () => {
```

Comment:
Nitpick: for consistency, can you make sure the `:` goes right after `interaction`?

Sidebar (not for this PR), I think anything `<category>: <test description>` should be using a `describe` block. The `:` is an indicator that the test is part of a group:

```ts
describe("keyboard interaction", () => {
  it("should change the value and emit input and change events", ...);
})
```